gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -235,6 +237,9 @@ def create_processed_variant(self, variant):
             proc_var, key, variant_info_data)
       else:
         proc_var._non_alt_info[key] = variant_info_data
+    for alternate_data in proc_var._alternate_datas:
```

Comment:
I think it's cleaner if this is moved to [here](https://github.com/googlegenomics/gcp-variant-transforms/blob/a15886c2a92353a5caed3d046c20bb3ba73bda1e/gcp_variant_transforms/libs/processed_variant.py#L407)
Basically, you'd add `alt_datas._annotation_field_names.add(annotation_field_name)` there.